### PR TITLE
Map Types

### PR DESCRIPTION
**Pull request by BootlegBow**
cant be fucked explaining, check the discord

this is probably a horrible way of coding it, if somebody wants to rework these as datums later go ahead
not tested

## Changelog
:cl:
add: maptypes
tweak: remora is now ship-type, icebox is now planetary
tweak: remorastation is now the NTSS remora and iceboxstation is now icebox facility 
/:cl:

### CODE REVIEW

**Comment by BootlegBow** [comment]
help

**Comment by BootlegBow** [comment]
@Kitsunemitsu fucking skyrat folder broke it

**Comment by Kitsunemitsu** [comment]
I will test merge on a local

**Comment by Kitsunemitsu** [comment]
FUCK FUCK FUCK

**Comment by Kitsunemitsu** [comment]
we have to fucking Atomize

**Comment by BootlegBow** [comment]
nice conflicts kirie

**Comment by Kitsunemitsu** [comment]
Same fucking shit

File: tgstation.dme [hunk 1]
```diff
@@ -3591,7 +3610,4 @@
 #include "ModularTegustation\zTegu_overwrite\cards.dm"
 #include "ModularTegustation\zTegu_overwrite\huds.dm"
 #include "ModularTegustation\zTegu_overwrite\tegu_overwrite_vars.dm"
-#include "modular_skyrat\modules\gunpoint\code\datum\gunpoint\gunpoint.dm"
```
Comment (Kitsunemitsu) [inline tgstation.dme:3594]: The Skyrat folder gives it's fucking vengence

**Pull request merged**